File: cross-application-feature-development-management/Directories/Scripts/EnvironmentVariablesSource/EnvironmentVariablesSourceDirectory.cs
namespace cross_application_feature_development_management.Directories.Scripts.EnvironmentVariablesSource
{
    public class EnvironmentVariablesSourceDirectory(
        ScriptsDirectory scriptsDirectory
        )
    {
        public string GetPath()
        {
            var environmentVariablesSourceDirectory =
                Path.Combine(scriptsDirectory.GetPath(), "environment-variables-source");

            return environmentVariablesSourceDirectory;
        }
    }
}

File: dotnet/cafdemvers/zzq-cafdemwimu/cafdemwimu.console/Directories/Repository/Dotnet/Scripts/EnvironmentVariablesSource/EnvironmentVariablesSourceDirectory.fs
namespace cafdemwimu.console.Directories.Repository.Dotnet.Scripts.EnvironmentVariablesSource

open System.IO
open cafdemwimu.console.Directories.Repository.Dotnet.Scripts

type EnvironmentVariablesSourceDirectory(scriptsDirectory: ScriptsDirectory) =
    member _.GetPath() =
        let environmentVariablesSourceDirectory =
            Path.Combine(scriptsDirectory.GetPath(), "environment-variables-source")
        environmentVariablesSourceDirectory
